docs/operator/deploy.md

```diff
@@ -231,13 +248,13 @@ For third party implementations of randomness beacon verification, you need:
 As an administrator of a drand node, you can use the control port to access the chain information:
 
 ```bash
-drand show chain-info
+drand show chain-info --id {beacon-id}
```

Comment:
Can we add the command to list all beacon IDs we have locally ? (do we have that ?)

Reply:
Done! :D